### ISSUE TITLE
Enum is being mapped with private member variable that cannot be ignored

### ISSUE THREAD

**Issue by mattdandrea** [xebia-france/selma#118]
I cannot seem to ignore the private "description" field from showing up in the generated Selma mapping class.
And, it is trying to access it like it's public for some reason:
```
   switch (in) {
        case ONE :  {
        ...
          break;
        }
        case TWO :  {
        ...
          break;
        }
        case THREE :  {
		...
          break;
        }
        case description :  {
		...
          break;
        }
      }
```
Any ideas?

Here is the enum (it matches for both the source and destination):

```
public enum NumberEnum {
	ONE("one"),
	TWO("two"),
	THREE("three");
	
	private String description;
	
	private NumberEnum(String description) {
		this.description = description;
	}
	
	public String getDescription() {
		return description;
	}
}
```

**Comment by slemesle** [comment]
Hi it looks like a bug, Selma should definitely ignore this kind of members. 
I'll have a deeper look on this quickly. 
Thanks for the report.

**Comment by mattdandrea** [comment]
Thanks for fixing this!  Will this only be available in the 1.0 release or before as well?

**Comment by slemesle** [comment]
Winter and 1.0 is coming :)
